### ISSUE TITLE
 Add support for Gnome 43.1 

### ISSUE THREAD

**Issue by spastorino** [pop-os/gnome-shell-extension-system76-power#85]
I'm using Arch Linux which is on Gnome 43.1 and this shell extension is not supported in 43.1. I've tried including the version in metadata.json, but that's not enough. It needs supporting code for it.

**Comment by avra-m3** [comment]
Gnome 43 removed aggregate menu in favor of quicksettings https://gjs.guide/extensions/upgrading/gnome-shell-43.html#quick-settings

**Comment by thekarananand** [comment]
talking in layman's terms, is it Possible? if Yes, is the team working on this?

**Comment by Eeems** [comment]
> talking in layman's terms, is it Possible? if Yes, is the team working on this?

Possible, yes. Are they working on it, :man_shrugging:
